RELEASE.md

```diff
@@ -1,15 +1,61 @@
+- Review the diff since the last release: `$ git diff main $(git log --grep '${PKGDIR}/version - ${CURRENT_VERSION}' --format=tformat:%h) packages/${PKGDIR}`
+- Determine the new version number based on the nature of changes
+  - Follow up with author or reviewers if the SemVer-nature of an included change is unclear before proceeding
 
-3) From `main` branch, run `yarn semantic-publish` in project root
+## 1. Preparing your release
+
+1) Bumping version
+  - Create a new branch: `git checkout -b release-${PKGDIR}-${VERSION}`
+  - Bump the `version` field in `packages/{PKGDIR}/package.json` in an individual commit
+    - Commit title must be of format `${PKGDIR}/version - ${VERSION}`
+    - Commit body must contain an exhaustive list of changes to the package since last release (commit hashes and/or merged PR numbers).
+  - Build, lint, and test the package
+2) Bumping dependents
+  - Identify any other packages in the monorepo that depend on the current version of this package and should have their versions bumped
+    - Useful: `$ find packages -maxdepth 2 -name package.json -exec jq '[.name, .dependencies["@lavamoat/aa"], .dependencies["@lavamoat/aa"]] | select(.[1] or .[2])|@tsv' {} -r \;`
+  - Perform updates of dependents if necessary in additional commit(s)
+  - Build, lint, and test any updated dependants
+    - Keep in mind that as long as the version range of the dependency is satisfied by the locally checked out package, internal dependencies will be linked to that. Otherwise, it's fetched from the configured registry like any other dependency.
+3) Review
+  - `git diff release-${PKGDIR}-${VERSION} main` should only contain version and dependency bumps in `package.json` files, and the corresponding changes in `yarn.lock`
+
+## 2. Submitting a release PR
+
+Open a new [pull request](https://github.com/LavaMoat/LavaMoat/compare) from your branch targeting `main`.
+
+The PR description should contain an exhaustive list of included changes - usually you can just copy the commit message verbatim.
+
+If there are particularities reviewiers should be aware of (urgency, manual testing steps), that should be mentioned in the description or in comments to the PR.
+
+Request review from relevant LavaMoat team members. Await approval.
+
+## 3. Publish to npmjs.com
+
+This part can be done by a different person.
+
+0. Prerequisites
+  - PR reviewed
+  - An NPM publish token
+  - Push permissions to https://github.com/LavaMoat/LavaMoat `main` branch
+  - Ensure you are doing this from a secure environment and handle the npm token with utmost care
+  - Fresh clone to ensure a clean and consistent state
+1. `$ git checkout main`
+2. `$ git merge --ff-only origin/release-${PKGDIR}-${VERSION}`
+3. `$ NODE_ENV=production yarn && yarn setup && yarn build`
+4. Bring `NPM_TOKEN` into current environment
+5. ???
```

Comment:
???

Reply:
Same as your point 1, addressed: https://github.com/LavaMoat/LavaMoat/pull/434#pullrequestreview-1228581315

Ie I need help to fill this in!